message/outbox_test.go

```diff
@@ -29,7 +29,8 @@ func TestOutbox(t *testing.T) {
 		provider := message.NewFakeProvider(t)
 		bcast := true
 
-		ob := message.NewOutbox(w, message.FakeValidator{RejectMessages: true}, queue, publisher, message.NullPolicy{}, provider, provider)
+		ob := message.NewOutbox(w, message.FakeValidator{RejectMessages: true}, queue, publisher,
+			message.NullPolicy{}, provider, provider, newOutboxTestJournal())
```

Comment:
Don't depend across test files like this. Move the function here, and just use a single variable in handler_integration_test.